lib/core/network/api/services/remote_config_service.dart

```diff
@@ -161,8 +161,8 @@ class RemoteConfigService {
         },
         "eosTestnet": {
           "name": "Jungle4 Testnet",
-          "endpoint": "https://jungle4.dfuse.eosnation.io",
-          "fastEndpoint": "https://jungle4.dfuse.eosnation.io",
+          "endpoint": "http://jungle.eosusa.io/",
```

Comment:
should we be changing these endpoints in firebase and not locally?

Reply:
I was changing them in firebase, but the local app doesn't update for whatever reason. 

The mobile phones were updating correctly